ts/modules/xrange.src.ts

```diff
@@ -269,6 +270,17 @@ seriesType<Highcharts.XRangeSeries>('xrange', 'column'
 
         /* eslint-disable valid-jsdoc */
 
+        /**
+         * @private
+         * @function Highcarts.seriesTypes.xrange#init
+         * @return {void}
+         */
+        init: function (this: Highcharts.XRangeSeries): void {
+            seriesTypes.column.prototype.init.apply(this, arguments as any);
+
+            this.options.stacking = false as any; // #13161
```

Comment:
`false` is not a valid value for stacking. Why not set it to `void 0` so we can avoid the `any` cast?

Reply:
I extracted this value from OHLC series. 

https://github.com/highcharts/highcharts/blob/b5ef0bcb2fcad2af185f8a5f2bd8450b76157f97/ts/parts/OHLCSeries.ts#L219

So I guess that (void 0) we should set this value in both series. correct?

Reply:
Yes, correct. This is a survivor from pre-TypeScript code. When converting to TypeScript, we systematically cast all inconsistencies like this to `any` to avoid altering the functionality.  But as we spot obvious mistakes like this, we should seek to remove the `any` casts while working on related parts of the code.

The final step in the TypeScript conversion will be to go over all `any` casts and refactor them.

Reply:
ok, so will commit changes in both files here.

Reply:
@TorsteinHonsi fixed, also in navigator and tests.